app/views/articles/_about_author.html.erb

```diff
@@ -6,7 +6,7 @@
   </div>
   <div class="main-content">
     <h4><a href="/<%= @user.username %>"><%= @user.name %></a><%= follow_button(@user) %></h4>
-    <p><%= @user.summary %></p>
+    <p><%= @user.summary_html&.html_safe %></p>
```

Comment:
Hi @bolariin, this is still not correct :) For users that don't have the `summary_html` this is going to generate an error. If you type `nil.html_safe` in your console you'll see.

You have to do the same thing everywhere: if there's a summary_html render that, otherwise render the summary.

Reply:
I'm correcting this now